solc.js

```diff
@@ -171,14 +166,15 @@ if (!options.basePath && options.includePath && options.includePath.length > 0)
   abort('--include-path option requires a non-empty base path.');
 }
 
-if (options.includePath)
-  for (const includePath of options.includePath)
-    if (!includePath)
-      abort('Empty values are not allowed in --include-path.');
+if (options.includePath) {
+  for (const includePath of options.includePath) {
+    if (!includePath) { abort('Empty values are not allowed in --include-path.'); }
+  }
+}
 
-var sources = {};
+const sources = {};
```

Comment:
let?

Reply:
I was also confused but apparently the initial type assignment is const and not the values of it 🤷

Reply:
wat ok

Reply:
See also https://github.com/ethereum/solc-js/pull/566#discussion_r790301655

Reply:
Yeap 

```js
// valid
const person = { name: "stephen" }
person.name = "leon"

// invalid
const personTwo = { name: "stephen" }
personTwo = ""

// valid 
let personThree = { }
personThree = ""

Reply:
> ``` javascript
> // valid
> const person = { name: "stephen" }
> person.name = "leon"
>
> // invalid
> const personTwo = { name: "stephen" }
> personTwo = ""
> ```

Yeah, I kinda expected that.

> ``` javascript
> // valid 
> let personThree = { }
> personThree = ""
> ```

wat

JS was supposed to be only the good parts now :P

Reply:
:D 

There is barely any reason to use let over const only in situations like

```js
let count = 0;
count += 1;
```